src/App.js

```diff
@@ -30,6 +31,7 @@ function App() {
         <Route exact path='/' component={Landing} />
         <Route exact path='/sponsor' component={Sponsor} />
         <Route path='/' component={Footer} />
+        <Route path='/attend' component={Attend} />
```

Comment:
Should also have the `exact` prop passed unless you actually want this component to be rendered on pages like `dragonhacks.org/attend/my/other/page`